tests/tests_unit/test_session.py

```diff
@@ -1,125 +1,60 @@
 from pathlib import Path
 
-import pandas as pd
+import pytest
 
-from cognite.neat._session import _state_machine as states
+from cognite.neat import _state_machine as states
 from cognite.neat._session._session import NeatSession
-from tests.config import DATA_FOLDER
+from tests.tests_unit.test_data_model.test_importers.test_dms_table_importer import valid_dms_yaml_formats
 
+session = NeatSession()
 
-def markdown_to_dict(file_path: Path) -> dict:
-    # Parse the markdown table into a pandas DataFrame
-    df = pd.read_csv(file_path, sep="|", skipinitialspace=True)
 
-    # Clean up the DataFrame by removing empty columns and whitespace
-    df = df.dropna(axis=1, how="all")  # Remove empty columns
-    df = df.apply(lambda x: x.str.strip() if x.dtype == "object" else x)  # Strip whitespace
+class TestNeatSession:
+    def test_read_data_model(self, tmp_path: Path) -> None:
+        read_yaml = tmp_path / "read.yaml"
+        read_yaml.write_text(next(iter(valid_dms_yaml_formats())).values[0])  # type: ignore [attr-defined]
```

Comment:
It is expensive to write and read to disk. This is part of the reason why the v0 test battery is slow. So suggest using a mock, see https://github.com/cognitedata/neat/blob/main/tests/tests_unit/test_data_model/test_importers/test_dms_table_importer.py#L1230

Currently, it takes ~10 second to execute all v1 tests. I think we should strive to have a fast 

<img width="764" height="730" alt="Image" src="https://github.com/user-attachments/assets/842fba7f-6335-4822-bc9e-004922abd250" />